### PR TITLE
Use camelcase for experiment/cookie names

### LINKED ISSUE #6821: Use camelcase for experiment/cookie names
See: https://github.com/mozilla/addons-frontend/pull/6675#discussion_r230092274

---

We should use camel case for the experiment names, instead of snake cases because we use camel case everywhere else.

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #6821

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -386,7 +386,7 @@ curl https://addons-dev.allizom.org/__frontend_version__
     "build": "https://circleci.com/gh/mozilla/addons-frontend/10333",
     "commit": "47edfa6f24e333897b25516c587f504e294e8fa9",
     "experiments": {
-        "home_hero": true
+        "homeHero": true
```
[the remark this answers is not in the document]
Comment (willdurand) [reply README.md]: mmh, this is the README file so it does not matter. Yet, the second link points to a variable with a value that should be updated.

[the remark this answers is not in the document]
Comment (rebmullin) [reply README.md]: sorry..I meant to ask about this in the default config

[the remark this answers is not in the document]
Comment (willdurand) [reply README.md]: fixed 😋

[the remark this answers is not in the document]
Comment (rebmullin) [reply README.md]: cool, thanks : )

**Pull request merged**